nada_numpy/array.py

```diff
@@ -948,6 +948,31 @@ def test_shuffle_randomness(vector_size, num_shuffles):
 
         return arr
 
+    def var(self) -> "NadaArray":
+        """
+        Compute the variable of array elements.
+
+        Returns:
+            nadaarray: the variance of array elements
+        """
+
+        # Compute the mean
+        m = self.mean()
+
+        # Initialise the sum to 0
+        sum = Integer(0)
```

Comment:
Nada Numpy also supports SecretRational types. The function should also work for Rational types. You can probably look into the implementation of other functions to check for this.